src/App.css

```diff
@@ -0,0 +1,18 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap');
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+  justify-self: center;
+  margin: 0;
```

Comment:
- [ ] Great job with the design. there is this left to right scroll. please fix that.
- [ optional] to make the app more user friendly, you can ensure that when clicked on the continent name and not only the arrow icon it also takes you to the detail section 
![demo2](https://user-images.githubusercontent.com/65298048/187805951-b6a64288-6d0c-47d5-b367-31ba37a62f42.gif)

Reply:
I have addressed the horizontal scroll issue by adding max-width: 100% and overflow-x: hidden rule sets to the html, body selectors